src/Raven.Studio/typescript/viewmodels/database/indexes/deleteIndexesConfirm.ts

```diff
@@ -3,35 +3,85 @@ import dialog = require("plugins/dialog");
 import database = require("models/resources/database");
 import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
 import messagePublisher = require("common/messagePublisher");
+import index = require("models/database/index/index");
+import indexDefinition = require("models/database/index/indexDefinition");
 
-class deleteIndexesConfirm extends dialogViewModelBase {
+class indexInfoForDelete {
+    indexName: string;
+    reduceOutputCollection: string;
+    ReferenceCollection: string;
 
-    deleteTask = $.Deferred<boolean>();
-    title: string;
+    readonly referenceCollectionExtension = "/References";
+    
+    constructor(name: string, reduceCollection: string, hasReferenceCollection: boolean) {
```

Comment:
can we name parameters same as props we assign to?

Reply:
done